odoo/models.py

```diff
@@ -5123,6 +5123,25 @@ def copy(self, default=None):
 
         return record_copy
 
+    @api.returns(None, lambda value: value.ids)
+    def copy_multi(self, default=None):
```

Comment:
IMO, we should modify the `copy` itself to be batchable. But I understand that this is completely out of your scope, and changing of this API will break the previous method signature.

We will try to do the latter (or maybe not, if it breaks too much).